bcipy/gui/BCInterface.py

```diff
@@ -327,6 +328,54 @@ def check_input(self) -> bool:
             return False
         return True
 
+    def user_id_validator(self) -> bool:
+        """Check User ID Requirements. User ID must follow the following Requirements:
+
+        1. Maximum length of 25 letters or numbers
+        2. Minimum length of at least one letter or  number
+        3. No special characters
+        4. No spaces
+        """
+
+        # Update based on current inputs
+        self.user = self.user_input.currentText()
```

Comment:
As mentioned below this value should already be set when you call this in `check_input()`. In the current flow, we would extract user from the input and assign self.user twice.

Reply:
deleted from function